### DISCUSSION TITLE
Consistent naming of app

### DISCUSSION THREAD

**Discussion by drawtheliner** [iSoron/uhabits#1377] [Feature Requests] [answered]
This isn't really an issue, but a suggestion. As far as I know, the oficial full name of the app is Loop Habit Tracker. I think it's safe to say that "Loop" is the main identifier here (also because the logo is a circle arrow).

However everywhere in Android the app shows with the generic name "Habits" - even at the top in the apps UI. Why is that?

I'm also confused about the name of this github rep - why is it uhabits?

Again, not a big deal. Just a bit confusing.

**Comment by iSoron** [comment, marked as answer]
I agree that we should have used "Loop" everywhere from the beginning. Unfortunately fixing it now could potentially cause some issues:

1. Current users already know that the app becomes "Habits" when installed; if we change it to "Loop", existing users may have issues finding it, or may think it got uninstalled.
2. We are currently the top non-sponsored search result for the keyword "habit" on Google Play. No one knows for sure, but changing the launcher name could plausibly affect our ranking. Our GitHub URL also ranks highly on Google and other search engines. Renaming the repository now might also affect this. The URL is also mentioned in many online posts.

**Reply by drawtheliner** [reply]
Got it! Makes sense.
